### DISCUSSION TITLE
How can I use one controller for multiple bots?

### DISCUSSION THREAD

**Discussion by MaratKarimov21** [telegram-bot-rb/telegram-bot#214] [General]
I want multiple bots to have same logic so can I use one controller for that? And I also need to add new bots dynamically, is this possible?

**Comment by printercu** [comment]
It's possible. See how routes helper is implemented https://github.com/telegram-bot-rb/telegram-bot/blob/master/lib/telegram/bot/routes_helper.rb#L41
You can have a similar one that will decide what bot to use.
